main.py

```diff
@@ -87,12 +87,20 @@ class WeatherStations(BaseModel):
     """
     # NOTE: percentile, start_year and end_year input is ignored, all responses overriden to match
     # pre-calculated values; 90th percentile, start year 2010, end year 2019
-    response = CalculatedResponse(percentile=90, year_range=YearRange(start=2010, end=2019))
+    default_year_range = YearRange(start=2010, end=2019)
```

Comment:
We actually no longer need a default year range - we can now make it required - and use the actual year range supplied. This was just a workaround I introduced before we had year ranges.

Reply:
Removed default year range.